src/operations/expansions/extrudePolygon.js

```diff
@@ -0,0 +1,32 @@
+const { mat4, vec3 } = require('../../math')
+
+const { geom3, poly3 } = require('../../geometry')
+
+// Extrude a polygon in the direction of the offsetvector.
+// Returns (geom3) a new geometry
+const extrudePolygon = (offsetvector, polygon1) => {
```

Comment:
This is unrelated to extrusions I am guessing ? Should it not be 'offsetPolygon' ? (just asking, drawing a blank)

Reply:
I’ll look again. It may be in the wrong place.

Reply:
extrudePolygon() is being used by expandShell() for geom3 expansion. it's included here as I found no other use of this function in the library.

Reply:
@kaosat-dev should I move extrudePolygon to extrusions?

Reply:
not sure to be honest : is it actual extrusion ? Ie like taking a flat shape and turning it 3D ?

Reply:
Correct. It uses the polygon as the top and the bottom, and builds sides. The result is a new 3D geometry.

Reply:
Let’s leave it here for now.